assets/terraform-modules/packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -131,6 +131,7 @@ systemd:
           --client-ca-file=/etc/kubernetes/ca.crt \
```

Comment:
I think this should be merged with commit `packet: add variable cloud_provider`.